successgilli/quick-credit
# #165993508 Bg: UI fixes pages

#### What does this PR do?
This PR restructured and fixes bugs in my front and server side
#### Description of Task to be completed?
- Make loan calculator into one container
- make close buttons sensitive without much precision
- Remove buttons from tables
#### How should this be manually tested?
Git clone the repo and cd into the server folder and run 'npm run startDev' which will listen on port 8080.
open postman and hit the routes and alson you can enter the UI folder to see changes in features.
#### Any background context you want to provide?
#### What are the relevant pivotal tracker stories?
#165993508
#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/42284402/58189579-aec9e280-7cb2-11e9-84c9-d53cc7a8912c.png)
![image](https://user-images.githubusercontent.com/42284402/58189690-e0db4480-7cb2-11e9-8333-1ea9ffb20ebb.png)
![image](https://user-images.githubusercontent.com/42284402/58189744-f6506e80-7cb2-11e9-843e-59b2978e2725.png)


#### Questions:
